src/app/forms/forms.module.js

```diff
@@ -1,5 +1,137 @@
 (function() {
   'use strict';
 
-  angular.module('uvpBuilderWeb.forms', []);
+  angular.module('uvpBuilderWeb.forms', ['flow', 'ui.router'])
+    .config(['flowFactoryProvider', function (flowFactoryProvider) {
```

Comment:
Remember to use `.$inject` and to define the provider creating a function instead of passing an anonymous function directly to `config`